README.md

```diff
@@ -151,6 +151,9 @@ artifactory-cleanup --output=myfile.txt
 
 # Save the summary in a Json file
 artifactory-cleanup --output=myfile.txt --output-format=json
+
+# Save the summary and removed artifacts list in a Json file
+artifactory-cleanup --output=myfile.txt --output-format=json --save-removed-artifacts
```

Comment:
The last argument looks like it doesn't affect `--output`, but it does.

We should name it in a more generic way, like 
```suggestion
artifactory-cleanup --output=myfile.txt --output-format=json --output-artifacts 
```